Makefile

```diff
@@ -107,6 +108,10 @@ run: ## start the development server
 	@$(COMPOSE) up -d app
 .PHONY: run
 
+node: ## an alias for "docker-compose run --rm node /bin/sh"
+	@$(COMPOSE_RUN) node /bin/sh
+.PHONY: node
```

Comment:
I would prefer having a `bin/node` shortcut script than defining Makefile rules for this type of tasks.

Reply:
Good Idea) Thanks) Added `bin/create-plugin` and `bin/run-plugin` scripts for plugins management) Replaced this Makefile rule with `build-node`  - to build the Node Docker image)